manticore/core/state.py

```diff
@@ -231,6 +231,19 @@ def solve_n(self, expr, nsolves, policy='minmax'):
         '''
         return self._solver.get_all_values(self.constraints, expr, nsolves, silent=True)
 
+    def solve_buffer(self, addr, nbytes):
+        '''
+        Reads `nbytes` of symbolic data from a buffer in memory at `addr` and attempts to
+        concretize it
+
+        :param int address: Address of buffer to conretize
+        :param int nbytes: Size of buffer to conretize
+        :return: Concrete contents of buffer
+        :rtype: list[int]
+        '''
+        buffer = self.cpu.read_bytes(addr, nbytes)
+        return ''.join(chr(self.solve_one(x)) for x in buffer)
```

Comment:
Why does it matter that the solutions are related to each other? I figure as long as a solution exists that satisfies the constraints, it is fine?